src/Sdk/WebApi/WebApi/ResultsHttpClient.cs

```diff
@@ -251,23 +345,22 @@ public async Task UploadStepSummaryAsync(string planId, string jobId, Guid stepI
             // Upload the file
             using (var fileStream = new FileStream(file, FileMode.Open, FileAccess.Read, FileShare.Read, 4096, true))
             {
-                var response = await UploadBlockFileAsync(uploadUrlResponse.SummaryUrl, uploadUrlResponse.BlobStorageType, fileStream, cancellationToken);
+                await UploadBlockFileAsync(uploadUrlResponse.SummaryUrl, uploadUrlResponse.BlobStorageType, fileStream, cancellationToken);
             }
 
             // Send step summary upload complete message
             await StepSummaryUploadCompleteAsync(planId, jobId, stepId, fileSize, cancellationToken);
         }
 
-        private async Task<HttpResponseMessage> UploadLogFile(string file, bool finalize, bool firstBlock, string sasUrl, string blobStorageType,
```

Comment:
we we no longer need to return ResponseMessage?

Reply:
Correct, we were never reading response anyway.  We always throw exceptions if something's wrong.